measures/ApplyUpgrade/tests/apply_upgrade_test.rb

```diff
@@ -465,17 +532,22 @@ def _test_heat_pump_backup(heat_pump_type, heat_pump_is_ducted, expected_values)
     end
   end
 
-  def _test_duct_restriction(upgrade_max_airflow_cfm, fan_watts_per_cfm, expected_values)
+  def _test_duct_restriction(heat_pump_sizing_is_duct_limited, upgrade_max_airflow_cfm, fan_watts_per_cfm, expected_values)
     this_dir = File.dirname(__FILE__)
     hpxml_path = File.join(this_dir, '../../UpgradeCosts/tests/in.xml')
     hpxml = HPXML.new(hpxml_path: hpxml_path)
 
+    runner = OpenStudio::Measure::OSRunner.new(OpenStudio::WorkflowJSON.new)
+    measures = { 'ResStockArguments' => [{ 'heat_pump_sizing_is_duct_limited' => heat_pump_sizing_is_duct_limited }],
+                 'BuildResidentialHPXML' => [{ 'heat_pump_type' => HPXML::HVACTypeHeatPumpAirToAir }] }
+
     # Create instance of the measure
     measure = ApplyUpgrade.new
 
     hpxml.buildings.each do |hpxml_bldg|
-      actual_values = measure.get_duct_restriction_values(hpxml_bldg)
-      baseline_max_airflow_cfm = actual_values['max_airflow_cfm']
+      baseline_max_airflow_cfm = measure.set_autosizing_limits(runner, measures, hpxml_bldg)
```

Comment:
Call this parent method instead. Then we can check BuildResidentialHPXML argument assignments.

Reply:
I can't replace calling `get_adjusted_fan_watts_per_cfm` with parent `set_adjusted_fan_efficiency` since we're not actually re-applying the BuildResidentialHPXML measure first.